packages/loader/container-utils/src/error.ts

```diff
@@ -48,6 +49,61 @@ export class DataCorruptionError extends LoggingError implements IErrorBase {
     }
 }
 
+/**
+ * Coerce the throwable input into a DataCorruptionError.
+ * @param error - Throwable input to be converted.
+ */
+export function CreateCorruptionError(
+    error: any,
+    info: Partial<{
+        clientId: string,
+        messageClientId: string,
+        sequenceNumber: number,
+        clientSequenceNumber: number,
+        referenceSequenceNumber: number,
+        minimumSequenceNumber: number,
+        messageTimestamp: number,
+    }> = {},
+): DataCorruptionError {
+    if (typeof error === "string") {
+        return new DataCorruptionError(error, { ...info });
+    } else if (!error || Array.isArray(error) || typeof error !== "object") {
```

Comment:
!error will catch false, but not true. What's the intention here?
If intention is to check for null or undefined, then we would rather make an explicit check here.

Reply:
The condition in aggregate intends to capture all "non-objects" in the `{}` sense (i.e. arrays, nulls, and and pretty much anything else).  Is the following any more satisfactory?

```suggestion
    } else if (error === null || Array.isArray(error) || typeof error !== "object") {
```